docs/getting-started/install-configure.md

```diff
@@ -201,29 +201,110 @@ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.
 </div>
 </div>
 
-## Select Provider
+## Select a Getting Started Configuration
+
+Crossplane goes beyond simply modelling infrastructure primitives as custom
+resources - it enables you to define new custom resources with schemas of your
+choosing. We call these "composite resources" (XRs). Composite resources compose
+managed resources -- Kubernetes custom resources that offer a high fidelity
+representation of an infrastructure primitive, like an SQL instance or a
+firewall rule.
+
+We use two special Crossplane resources to define and configure these new custom
+resources:
+
+- A `CompositeResourceDefinition` (XRD) _defines_ a new kind of composite
+  resource, including its schema. An XRD may optionally _offer_ a claim (XRC).
+- A `Composition` specifies which resources a composite resource will be
+  composed of, and how they should be configured. You can create multiple
+  `Composition` options for each composite resource.
+
+XRDs and Compositions may be packaged and installed as a _configuration_. A
+configuration is a [package] of composition configuration that can easily be
+installed to Crossplane by creating a declarative `Configuration` resource, or
+by using `kubectl crossplane install configuration`. In the examples below we
+will install a configuration that defines a new `CompositePostgreSQLInstance` XR
+that takes a single `storageGB` parameter, and creates a connection `Secret`
+with keys for `username`, `password`, and `endpoint`. A `Configuration` exists
+for each provider that can satisfy a `PostgreSQLInstance`. Let's get started!
```

Comment:
```suggestion
for each provider that can satisfy a `CompositePostgreSQLInstance`. Let's get started!
```

This is the first time we mention the claim type and readers don't know where it comes from yet. We can stick to `CompositePostgreSQLInstance` until we talk about claims or maybe mention the claim concept here.